styles.css

```diff
@@ -1 +1,70 @@
+html {
+    background-color: pink;
+    
+    font-family: Georgia, 'Times New Roman', Times, serif;
+}
 
+.container {
+    margin: 100px 100px 100px 100px;
+    background-color: white;
+    background: url(background.jpg) no-repeat;
+    background-size: 1280px 768px;
+    width: 1280px;
```

Comment:
Shouldn't use the static width and height. It will make responsive is hard to use in the future.